data/src/message.rs

```diff
@@ -544,12 +544,24 @@ impl Limit {
     }
 }
 
-fn is_action(text: &str) -> bool {
-    text.starts_with("\u{1}ACTION ") && text.ends_with('\u{1}')
+pub fn ctcp_command(text: &str) -> Option<&str> {
+    let command = text
+        .strip_suffix('\u{1}')
+        .unwrap_or(text)
+        .strip_prefix('\u{1}')?;
+
+    command.split_whitespace().next()
+}
```

Comment:
I think we have some logic in the dcc module for parsing ctcp payload as well. Can we reference this from that module so we don't have duplicated logic for doing this?

Reply:
Sounds good.  I'll consolidate everything in a CTCP module.